packages/graphql/src/schema-model/annotation/AuthorizationAnnotation.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const AuthorizationAnnotationArguments = {
+    filter: "filter",
+    validate: "validate",
+};
```

Comment:
I find this an odd export; it's only used in one place for logging the values, so this could well be an array?

Reply:
You're right, changed, no reason, we changed this file over multiple iterations and now it could be better typed as a simple array!